apps/remix/app/root.tsx

```diff
@@ -15,73 +16,15 @@ import {
 import launchpadStyles from '@launchpad-ui/tokens/index.css';
 import themeStyles from '@launchpad-ui/tokens/themes.css';
 import globalStyles from './styles/global.css';
-
-// plop start imports
-import alertStyles from '@launchpad-ui/alert/style.css';
-import avatarStyles from '@launchpad-ui/avatar/style.css';
-import bannerStyles from '@launchpad-ui/banner/style.css';
-import buttonStyles from '@launchpad-ui/button/style.css';
-import chipStyles from '@launchpad-ui/chip/style.css';
-import clipboardStyles from '@launchpad-ui/clipboard/style.css';
-import counterStyles from '@launchpad-ui/counter/style.css';
-import drawerStyles from '@launchpad-ui/drawer/style.css';
-import filterStyles from '@launchpad-ui/filter/style.css';
-import formStyles from '@launchpad-ui/form/style.css';
 import iconStyles from '@launchpad-ui/icons/style.css';
-import markdownStyles from '@launchpad-ui/markdown/style.css';
-import menuStyles from '@launchpad-ui/menu/style.css';
-import modalStyles from '@launchpad-ui/modal/style.css';
-import navigationStyles from '@launchpad-ui/navigation/style.css';
-import paginationStyles from '@launchpad-ui/pagination/style.css';
-import popoverStyles from '@launchpad-ui/popover/style.css';
-import progressBubblesStyles from '@launchpad-ui/progress-bubbles/style.css';
-import progressStyles from '@launchpad-ui/progress/style.css';
-import selectStyles from '@launchpad-ui/select/style.css';
-import sliderStyles from '@launchpad-ui/slider/style.css';
-import snackbarStyles from '@launchpad-ui/snackbar/style.css';
-import splitButtonStyles from '@launchpad-ui/split-button/style.css';
-import tabListStyles from '@launchpad-ui/tab-list/style.css';
-import tableStyles from '@launchpad-ui/table/style.css';
-import tagStyles from '@launchpad-ui/tag/style.css';
-import toastStyles from '@launchpad-ui/toast/style.css';
-import toggleStyles from '@launchpad-ui/toggle/style.css';
-import tooltipStyles from '@launchpad-ui/tooltip/style.css';
-// plop end imports
```

Comment:
Huge DX win for Remix integration!

Reply:
@Niznikr I was thinking of proposing this in gonfalon, to add an eslint rule to default to named exports, any thoughts on this?

Reply:
that's what I was thinking, on all new additions as to not be a pain, or manually add an ignore rule statement on the top so we can remember to peel them off incrementally! thanks for your advice!

Reply:
👋 In general I'm a fan of using named exports anywhere we can... to y'alls point(s) above we can add exclusions for valid exceptions as necessary if we choose to go this route.

Sorta related, we've used code-mods a couple of times now to bulk-exclude existing violations of a new lint rule while also turning it on, to be able to prevent the spread while also reducing dev friction ([example PR](https://github.com/launchdarkly/gonfalon/pull/27440/files).

Reply:
thanks for the example P.R, this is very useful to know. I've document this effort in a shortcut [ticket](https://app.shortcut.com/launchdarkly/story/205508/create-eslint-rule-to-enforce-named-exports)